Neptune/Services/IService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Net.Http;
+
+namespace Neptune
+{
+    public interface IService { }
```

Comment:
Oh this class does what I thought you were doing in the first place. This one is nice to have